typescript/infra/scripts/core.ts

```diff
@@ -28,16 +28,20 @@ async function main() {
 
   let previousContracts = {};
   previousAddressParsing: try {
-    if (environment === 'test') {
+    if (environment === 'test' && 1 + 2 === 2) {
```

Comment:
You'll wanna remove this

Reply:
haha yeah still am making some changes, will rm when done testing things